src/App.js

```diff
@@ -42,45 +53,22 @@ export default class App extends React.Component {
 				<GlobalStyle />
 				<AppContainer>
 					<NavBar />
-					<HistorysContainer>
-						<History />
-					</HistorysContainer>
-					<PostsContainer>
-						{this.state.photos.length > 0 &&
-							this.state.photos[0].map((photo) => {
-								return (
-									<PostContainer>
-										<div className='post-header'>
-											<HeaderStatus>
-												<div className='profile-image-container'>
-													<img
-														src={photo.user.profile_image.medium}
-														alt='user-profile'
-														className='profile-image'
-													/>
-												</div>
-												<div className='status'>
-													<h4>{photo.user.name}</h4>
-													<h4 className='posted-time'>12 hour ago</h4>
-												</div>
-											</HeaderStatus>
-											<More>
-												<MoreIcon />
-											</More>
-										</div>
-
-										<p>{photo.user.bio}</p>
-										<div className='image-container'>
-											<img src={photo.urls.regular} alt='posted-img' />
-										</div>
-										<div className='Like-star'>
-											<Likes />
-											<Star />
-										</div>
-									</PostContainer>
-								);
-							})}
-					</PostsContainer>
+					<Switch>
+						<Route
+							exact
+							path='/search/collections/:id'
+							render={(props) => {
+								return <SearchCollection {...props} />;
+							}}
+						/>
+						<Route
+							exact
+							path='/'
+							render={() => {
+								return <PostPage photos={this.state.photos} />;
+							}}
+						/>
```

Comment:
move this route up (should be first)

And the data fetching should happen inside the PostPage

Rename the Post Page to Home